src/main/java/com/example/demo/entity/TicketType.java

```diff
@@ -0,0 +1,15 @@
+package com.example.demo.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.math.BigDecimal;
+
+@AllArgsConstructor
+@Getter
+public enum TicketType {
+    CHILD(BigDecimal.valueOf(12.50)),
```

Comment:
jak przyjdzie biznes i powie, że inflacja i trzeba podbić ceny to z czym się to będzie dla nas wiązało jako developerów?

Reply:
Ze zmianą ceny biletów w kodzie w tym miejscu i restartem aplikacji